server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -455,4 +558,44 @@ export class TenantManager {
         const accessInfo = JSON.parse(this.secretManager.decryptSecret(encryptedAccessInfo));
         return accessInfo;
     }
+
+    private decryptCachedKeys(cachedKey: string) {
+        const keys = JSON.parse(cachedKey);
+        return keys.key2 === "" ? {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: "",
+            } : {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: this.secretManager.decryptSecret(keys.key2),
+            };
+    }
+
+    private async getCacheKey(tenantId: string): Promise<string> {
+        return this.cache?.get(`tenantKeys:${tenantId}`);
+    }
+
+    private async deleteCacheKey(tenantId: string): Promise<boolean> {
```

Comment:
nit: would `deleteKeyFromCache` or even `deleteCachedKey` be more appropriate? Feel free to dismiss this.